hw/lint/Makefile

```diff
@@ -13,7 +13,7 @@
 
 CORE_ROOT ?= ../../
 REPORT_DIR ?= reports
-
+FSEL ?= ip
```

Comment:
just a note here that the Makefile flow is something that will be deprecated soon. but thanks for fixing it as well...

Reply:
yeah, figured might as well :)